hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1523,6 +1529,24 @@ message CancelPrepareResponse {
 
 }
 
+message ListOpenFilesRequest {
+  optional string path = 1;
```

Comment:
is it really optional ...? As far as I can tell the OM code expects these three parameter all the time.

Reply:
It is for proto3 migration at some point. Same idea as this:

https://github.com/apache/ozone/pull/3824#discussion_r1011376445